### ISSUE TITLE
about detaching

### ISSUE THREAD

**Issue by XiaoyuShi97** [princeton-vl/RAFT#116]
https://github.com/princeton-vl/RAFT/blob/aac9dd54726caf2cf81d8661b07663e220c5586d/core/raft.py#L123
Hi, I find that you detach coords before sampling. Is there any specific reason about this design choice? Thx!

**Comment by zachteed** [comment]
Hi, calling .detach() is quite important. Results are worse if this is not done and training less stable. This implementation detail is mentioned in Sec. 4 of our paper
